templates/image-formatter.html.twig

```diff
@@ -12,10 +12,13 @@
  * @see template_preprocess_image_formatter()
  *
  * @ingroup themeable
+ *
+ * This was fixed in https://www.drupal.org/project/drupal/issues/2829990. Once that commit is released, this template
```

Comment:
What about https://www.drupal.org/project/drupal/issues/3064751?

Reply:
Oh, I see the template below. Please ignore.

Reply:
that ones documented in the `responsive-image-formatter.html.twig` template

Reply:
oh beat me to the comment.